lib/mongoose_push/logger.ex

```diff
@@ -0,0 +1,25 @@
+defmodule MongoosePush.Logger do
+  @moduledoc """
+  Module responsible for formatting logger output
+  """
+  def format_fmt(level, message, {date, time}, metadata) do
+    date_f = Logger.Formatter.format_date(date)
+    time_f = Logger.Formatter.format_time(time)
+    meta_f = Logfmt.encode(metadata)
+
+    "\n#{date_f} #{time_f} [#{level}] #{meta_f} #{message}"
```

Comment:
The whole message should be encoded in the same format, there is very little use if only part of it is standardized.

Reply:
Hmm, although for logfmt specifically we may do this differently to make it more readable - but this exception I would apply at most for date and log level - message should be metadata.

Reply:
What exactly do we want to pass as a message then? Do we want it to have a specific format or should it just be a text description?

Reply:
Message should just be part of `metadata` map under key `msg` or something like that.
The only special treatment it can get is to position it "better". Please remember that `logfmt` does have positional formatting, unlike `json`.

Reply:
My doubts were about what to put as a message and what as metadata when calling `Logger.log` - the logger docs mentioned above have fields like `what` , `at` and `result` which cover the log message that existed before completely.

Reply:
I think in this naming, `what` == `msg`, at least I think so. Regardless of the name of this field - Elixir's Logger requires the 1st argument to be a string, and thus we have to put the message there. Everything else as metadata. On formatter level we can however merge those two and do whatever we feel like it.